core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -653,7 +657,23 @@ public SystemSessionProperties(
                                 getTimeZoneKey(value);
                             }
                         },
-                        true));
+                        true),
+                enumProperty(
+                        REGEX_LIBRARY,
+                        "Regexp library",
+                        RegexLibrary.class,
+                        featuresConfig.getRegexLibrary(),
+                        false),
+                integerProperty(
+                        RE2J_DFA_STATES_LIMIT,
+                        "RE2J DFA states limit",
+                        featuresConfig.getRe2JDfaStatesLimit(),
+                        false),
+                integerProperty(
+                        RE2J_DFA_RETRIES,
+                        "RE2J DFA retries",
+                        featuresConfig.getRe2JDfaRetries(),
+                        false));
```

Comment:
Adding session properties for these settings seem unrelated to this change.

Reply:
I've added these session properties because I couldn't access `featureConfig` easily.
Is it ok if I just create a dedicated commit for this change?